sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer

```diff
@@ -16,17 +16,15 @@ json_extract('[10, 20, [30, 40]]', '$[1]', '$[0]')
 ===================================================
 0
 ===================================================
-json_extract(@j1, '$.id', '$.name[0]', '$.name[1][1]', '$.name_old')    
-["109","hhh1","hhh3","hhh6"]     
-
+Error:-1200
```

Comment:
I think it's a bug. Please open an issue. 
Test Build: 10.2.0.8389-9cec2c6
Actual Result:
```
csql> select json_extract('{"name_old":"hhh6"}', '$.name_old');                                                                                                                                                   
ERROR: Invalid JSON path


0 command(s) successfully processed. 
```
Expected Result:
Same as 
```
csql> select json_extract('{"name_old":"hhh6"}', '$."name_old"');

=== <Result of SELECT Command in Line 1> ===

  json_extract('{"name_old":"hhh6"}', '$."name_old"')
======================
  "hhh6"              

1 row selected. (0.009353 sec) Committed.

1 command(s) successfully processed.
```